src/uct/tcp/tcp.h

```diff
@@ -280,13 +280,30 @@ typedef struct uct_tcp_ep_zcopy_tx {
 } uct_tcp_ep_zcopy_tx_t;
 
 
+/**
+ * TCP iface address flags
+ */
+typedef enum uct_tcp_iface_addr_flags {
+    UCT_TCP_IFACE_ADDR_FLAG_LOCAL = UCS_BIT(0)
```

Comment:
could you please document the meaning of this flag?

Reply:
done